MetaMask/metamask-extension
# Bug 17606 - Ledger locked message

## Explanation

Vague error message for connecting to Ledger devices that are locked.

## Screenshots/Screencaps
### Before

Showing "Ledger device: UNKNOWN_ERROR (0x6b0c)"
Other users report: "Ledger device: UNKNOWN_ERROR (0x5515)"

![image](https://user-images.githubusercontent.com/82925669/217112297-d59e580c-08c4-41c8-a87d-1aeee6a5dac7.png)

### After

Now showing the localized `ledgerLocked` message for `0x6b0c` error code (should also detect `0x5515`).

![image](https://user-images.githubusercontent.com/82925669/217112178-a22ca43e-eca8-422f-914e-ba8547b54c28.png)

## Manual Testing Steps

- attach ledger device
- either do not unlock, or wait for screensaver (floating icons) to appear
- attempt to connect ledger hardware wallet
- error should display with localized `ledgerLocked` message

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
